### PR TITLE
[13.0][IMP] multicompany_property_product: handle partner pricelist in partner properties

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Technical review, LTGM

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM!

File: multicompany_property_product/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class PartnerProperty(models.TransientModel):
+    _inherit = "res.partner.property"
+
+    parent_id = fields.Many2one(related="partner_id.parent_id")
+    property_product_pricelist = fields.Many2one(
```
Comment (MiquelRForgeFlow) [inline multicompany_property_product/models/res_partner.py:11]: I think you don't need to do this kind of thing, as thins kind of thing is done only for company_dependent fields.

What you need to do is only call the method _compute_product_pricelist (which uses force_company) with super and make the properly fix in that method.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: multicompany_property_product/models/res_partner.py (hunk 1 above)
Comment (JordiMForgeFlow) [reply multicompany_property_product/models/res_partner.py:11]: @MiquelRForgeFlow the problem is that, although it is not technically a company dependent field, logically it is as it ends up being managed as a property. Users that use these multicompany modules expect to handle all company dependent fields via the properties as it helps them see the values of the different companies and update them.

As you correctly say the field is handled via the compute method, that is why I don't get/write the value of the pricelist in the same way as we do for other fields in the properties.

Comment (MiquelRForgeFlow) [reply multicompany_property_product/models/res_partner.py:11]: Ok.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**